core/src/main/scala/sbt/librarymanagement/MakePomConfiguration.scala

```diff
@@ -0,0 +1,22 @@
+package sbt.librarymanagement
+
+import java.io.File
+import scala.xml.{ Node => XNode, NodeSeq }
+
+// final class DeliverConfiguration(
+//     val deliverIvyPattern: String,
+//     val status: String,
+//     val configurations: Option[Vector[Configuration]],
+//     val logging: UpdateLogging
+// )
+
+final class MakePomConfiguration(
+    val file: File,
+    val moduleInfo: ModuleInfo,
+    val configurations: Option[Vector[Configuration]] = None,
+    val extra: NodeSeq = NodeSeq.Empty,
+    val process: XNode => XNode = n => n,
+    val filterRepositories: MavenRepository => Boolean = _ => true,
```

Comment:
Is it an inclusion or exclusion filter? I think this class could benefit from some more documentation.